docs/learn/dealing-with-latency.md

```diff
@@ -296,7 +294,10 @@ Advanced games will have most of their world predicted, allowing the client and
 <!-- TODO add diagram examples (stun grenade for example) and flow of reconciliation -->
 
 :::info
-There's no prediction implementation right now in Netcode for GameObjects, but you can implement your own. See our [roadmap](https://unity.com/roadmap/unity-platform/multiplayer-networking) for more information.
+While Netcode for GameObjects does not have a full implementation of Client Side Prediction and Reconciliation, you can build Client Side Prediction and Reconciliation on top of our existing Client Side Anticipation building-blocks, `AnticipatedNetworkVariable` and `AnticipatedNetworkTransform`. These components allow differentiating between the "authoritative" value and the value that is shown to the players. These components provide most of the information needed to implement prediction, but do require you to implement certain aspects yourself. Because of the complexity inherent in building a full client prediction system, the details of that are left as an exercise for the user, and we recommend only advanced users pursue this option.
```

Comment:
> Because of the complexity inherent in building a full client prediction system, the details of that are left as an exercise for the user

Not sure I'd use "Because" here, there's not really a cause and effect.

Reply:
I'd maybe try and list some of the complexity. As a sort of "here be dragons". Just so users don't waste time starting things they don't realize is more complex than they thought.

Reply:
+1
Even a "warning" callout box that implementing client prediction is a complex process and that while the client anticipation feature provides the framework to implement such a system the remaining code required is for advanced netcoders.